src/js/entities-service/entities/patients.js

```diff
@@ -52,6 +52,18 @@ const _Model = BaseModel.extend({
     const workspacePatient = Radio.request('entities', 'get:workspacePatients:model', this.id);
     return workspacePatient.get('status');
   },
+  toggleActiveStatus() {
+    const workspacePatient = Radio.request('entities', 'get:workspacePatients:model', this.id);
```

Comment:
Let's remove the `getStatus` function and just do `getWorkspacePatient` for a single way to look it up on the patient

Reply:
Done 👍

Reply:
I mean can we replace the `getStatus()` function from this entity up on 53 and make a `getWorkspacePatient`?

Reply:
I believe that's the update I made... this `/entities/patients.js` file no longer has the `getStatus()` function. And I created the `getWorkspacePatient()` and use that instead of `patient.getStatus()`.